README.md

```diff
@@ -33,5 +33,5 @@ It is a general purpose statically typed, easy to learn and use language.
 #### Manuals:
 - [User Manual](https://jsmith1024.github.io/bill/userman/html/index.html)
 - [Reference Manual](https://jsmith1024.github.io/bill/refman/html/index.html)
-- [Bill_Primer](https://jsmith1024.github.io/bill/langman/html/index.html)
-- [Standard_Library_Refence](https://jsmith1024.github.io/bill/stlman/html/index.html)
+- [Bill Primer](https://jsmith1024.github.io/bill/langman/html/index.html)
```

Comment:
Are these links still correct?

Reply:
yes

Reply:
I don't see how they can work when the top level directory does not have directories "userman" "refman" and "langman" on the branch.

Reply:
Oh, I overlooked that...

Reply:
Resolving conversation and creating a new one because the code is detached from the conversation.